client/src/env.ts

```diff
@@ -8,21 +8,25 @@ export const DISABLE_AUTH = Boolean(
   JSON.parse(process.env.REACT_APP_DISABLE_AUTH || "false")
 );
 
-export const CRUD_MODE = Boolean(
-  JSON.parse(
-    process.env.REACT_APP_CRUD_MODE ||
-      JSON.stringify(process.env.NODE_ENV === "development")
-  )
+export const WRITER_MODE = Boolean(
+  JSON.parse(process.env.REACT_APP_WRITER_MODE || "false")
```

Comment:
How about we default to `import.meta.url.contains('@mdn/yari')` to distinguish whether we're in the context of mdn/content or mdn/yari?

Reply:
I don't think that'll work, since we inline the values of env vars at build time:
https://github.com/mdn/yari/blob/d3f7ec15aeb45145849683116ccda7455897ffa1/client/config/webpack.config.js#L617

Reply:
Of course we inline the env var, not the `const` in `env.ts` - still, since we set that variable when building the npm package, we don't need to do need to do this.